internal/model/label.go

```diff
@@ -7,3 +7,20 @@ type Label struct {
 	URL         string `json:"url"`
 	Color       string `json:"color"`
 }
+
+var AllLabels = []Label{
+	CommunityLabel,
+	BetaLabel,
+}
+
+var CommunityLabel Label = Label{
+	Name:        "Community",
+	Description: "This plugin is maintained by the Open Source Community",
+	URL:         "https://mattermost.com/pl/default-community-plugins",
+}
+
+var BetaLabel Label = Label{
+	Name:        "Beta",
+	Description: "This plugin is currently in Beta and it's not recommended in production use.",
```

Comment:
```suggestion
	Description: "This plugin is currently in beta and is not recommended for use in production.",
```

Reply:
I've changed it to:
`This plugin is currently in Beta and is not recommended for use in production.`

`Beta` is written with a capital `B` in the documentation (e.g. in https://docs.mattermost.com/process/definitions.html) Hence, I would also use a capital `B` here.